include/sys/zio_impl.h

```diff
@@ -24,7 +24,7 @@
  */
 
 /*
- * Copyright (c) 2013 by Delphix. All rights reserved.
+ * Copyright (c) 2012, 2015 by Delphix. All rights reserved.
```

Comment:
Can you clarify who "We" is (Delphix, Illumos, OpenZFS) in that policy? And you don't want to connect those years with a dash (e.g. 2012-2015)?